test/kino_flame/runner_cell_test.exs

```diff
@@ -80,6 +125,40 @@ defmodule KinoFLAME.RunnerCellTest do
                )\
                """
     end
+
+    test "restores Kubernetes source code from attrs" do
+      attrs = %{
+        "backend" => "k8s",
+        "name" => "my_runner",
+        "min" => 2,
+        "max" => 3,
+        "max_concurrency" => 15,
+        "runner_pod_template" => "some_template"
+      }
+
+      {_kino, source} = start_smart_cell!(RunnerCell, attrs)
+
+      assert source ==
+               """
```

Comment:
Nitpick: you can use `~s'''`, so there's no need for escaping `"""` in the string :)

Reply:
Oh right. nice.